functions/__tests__/endpoints/resend/index.test.js

```diff
@@ -1,7 +1,7 @@
 const testConfig = require("firebase-functions-test")();
 const request = require("supertest");
 const axios = require("axios");
-const { app } = require("../../../controllers/expressExample/index");
+const { app } = require("../../../controllers/endpoints/index");
```

Comment:
Instead of calling the folder "endpoints", let's rename it to "auth" since that's what this specific express app is referred as in deployment.